target/linux/ramips/image/mt7621.mk

```diff
@@ -529,6 +529,22 @@ define Device/cudy_x6
 endef
 TARGET_DEVICES += cudy_x6
 
+define Device/dlink_dap-1620-b1
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DAP-1620
+  DEVICE_VARIANT := B1
+  DEVICE_PACKAGES := kmod-mt7615e kmod-mt7615-firmware rssileds
```

Comment:
`kmod-mt7615e` is not needed here, `kmod-mt7615-firmware` already depends on it.